tests/actions.go

```diff
@@ -252,6 +260,11 @@ func (oa *operatorActions) CleanTidbCluster(info *TidbClusterInfo) error {
 		}
 	}
 
+	_, err := oa.kubeCli.CoreV1().Pods(info.Namespace).Get(getBackupDirPodName, metav1.GetOptions{})
```

Comment:
helm del won't delete these pods?

Reply:
this pod is create for get dirs, not manager by helm